### DISCUSSION TITLE
Easy "Set-Cookie" methods

### DISCUSSION THREAD

**Discussion by pfk84** [clue/framework-x#145] [Ideas]
I'd suggest adding something like [hansott/psr7-cookies](https://github.com/hansott/psr7-cookies) for easy cookie handling without having to manually fiddle with adding "Set-Cookie" headers...

Sadly, there is no withCookies() method defined in PSR-7, only withCookieParams() for the request...

**Comment by clue** [comment]
@pfk84 Interesting suggestion! I agree having helpers such as the package above available can help in applications that need more control over setting HTTP cookies. In either case, we may want to add a chapter about HTTP cookies to our documentation to at least give some basic pointers :+1: PRs welcome if this is something you want to look into :shipit:
